docs/en/ingest-management/release-notes/release-notes-8.1.asciidoc

```diff
@@ -48,6 +74,43 @@ that no validation is performed on this field. {agent-pull}30386[#30386]
 
 Review important information about the {fleet} and {agent} 8.1.1 release.
 
+[discrete]
```

Comment:
This section has already been reviewed in https://github.com/elastic/observability-docs/pull/1780.